imap_processing/tests/hit/test_message_log.py

```diff
@@ -0,0 +1,24 @@
+import bitstring
+import pandas as pd
+
+from imap_processing import decom, imap_module_directory
+from imap_processing.hit.l0.data_classes.message_log import MessageLog
+
+
+def test_message_log():
+    # HIT is referring to SHCOARSE as SC_TICK, so I manually updated the name of that
+    # field in the validation data from SC_TICK -> SHCOARSE.
+    # No other changes to the data were made.
+    test_file = imap_module_directory / "tests/hit/test_data/msglog_sample.ccsds"
+    validation_file = (
+        imap_module_directory / "tests/hit/validation_data/msglog_sample_raw.csv"
+    )
+    xtce_file = imap_module_directory / "hit/packet_definitions/P_HIT_MSGLOG.xml"
+
+    validation_data = pd.read_csv(validation_file)
+    packets = decom.decom_packets(test_file.resolve(), xtce_file.resolve())
+
+    for pkt_idx, packet in enumerate(packets):
+        msg_log = MessageLog(packet, "0.0", "mslog_sample.ccsds")
+        assert msg_log.SHCOARSE == validation_data["SHCOARSE"][pkt_idx]
+        assert msg_log.TEXT == bitstring.Bits(hex=validation_data["TEXT"][pkt_idx]).bin
```

Comment:
Some questions here: Do you want the message log as a string of binary 0s and 1s? Or do you want that as a bytes-like object? `b"TEXT"` or something encoded/decoded with ascii ("TEXT".encode("ascii"))

I'm wondering if you actually want to avoid bitstring here and compare against the specific validation_data provided rather than manipulating that?

Reply:
good question. Curious to know what this does
```
bitstring.Bits(hex=validation_data["TEXT"][pkt_idx]).bin
```

Reply:
Eric Christian said the messages could be left in binary for the L1A product. The validation data is in hex so I used the Bits class to convert to binary (there might be better way to do it?). I could double check with them on how they want the data stored. This is one of those products that doesn't really make sense for us to be producing an L1A file for, so maybe I should push back on this product as a whole?